pypeit/wavecalib.py

```diff
@@ -179,10 +179,25 @@ def build_waveimg(self, tilts, slits, spat_flexure=None):
                 Image holding tilts
             slits (:class:`pypeit.slittrace.SlitTraceSet`):
             spat_flexure (float, optional):
```

Comment:
Why are spatial and spectra flexure treated differently in the input arguments and documentation? It should be symmetric.